app/gui/suggestion-database/src/lib.rs

```diff
@@ -161,7 +161,8 @@ impl QualifiedNameToIdMap {
 /// displays all Constructors and Methods defined for this type.
 #[derive(Clone, Debug, Default)]
 struct HierarchyIndex {
-    inner: HashMap<entry::Id, HashSet<entry::Id>>,
+    inner:   HashMap<entry::Id, HashSet<entry::Id>>,
+    orphans: HashMap<QualifiedName, HashSet<entry::Id>>,
```

Comment:
The `orphans` field should be described in docs: What it contains, and why we need it.